list.txt

```diff
@@ -9255,7 +9233,6 @@ dynabird.com
 dynainbox.com
 dynastyantique.com
 dynofusion-developments.com
-dynu.net
```

Comment:
why do you remove it?

Reply:
add new commit - where SLD domain allowed